logstash-core/src/test/java/org/logstash/instrument/metrics/UptimeMetricTest.java

```diff
@@ -0,0 +1,63 @@
+package org.logstash.instrument.metrics;
+
+import org.junit.Test;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.time.temporal.ChronoUnit;
+
+import static org.junit.Assert.*;
```

Comment:
we should avoid as much as possible multi includes. Probably it is because of your IntellJ setting. I always set it to 999 to avoid this issue :)